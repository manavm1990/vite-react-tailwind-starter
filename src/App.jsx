export default function App() {
  return <h1 className="font-black my-3 text-8xl text-center">Hello World</h1>;
}
