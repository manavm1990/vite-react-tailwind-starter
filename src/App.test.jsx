import renderer from "react-test-renderer";
import { it } from "vitest";
import App from "./App";

function validateTree(component) {
  const result = component.toJSON();
  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);
  return result;
}

it("renders correctly", () => {
  const tree = validateTree(renderer.create(<App />));
  expect(tree).toMatchSnapshot();
});
