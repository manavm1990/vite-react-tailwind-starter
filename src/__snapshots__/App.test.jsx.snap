// Vitest Snapshot v1

exports[`renders correctly 1`] = `
<h1
  className="font-black my-3 text-8xl text-center"
>
  Hello World
</h1>
`;
